ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -363,9 +362,9 @@ export default defineComponent({
         maxLength(70)
       ),
       phoneNumberRules: customRules(
-        required('Enter a phone number'),
-        maxLength(15)
-      )
+        isPhone(14, null)
```

Comment:
No need for a second argument, as we already have a default value of null.

Reply:
Good point... Changed.